src/components/QRCodeReader.js

```diff
@@ -111,7 +112,7 @@ export default ({
         borderLeftWidth: borderWidth,
         borderRightWidth: borderWidth,
         backgroundColor: 'transparent',
-        borderColor: 'rgba(0, 0, 0, 0.7)',
+        borderColor: COLORS.textColorShadowDark,
```

Comment:
Change from `0.7` opacity to `0.8` opacity

Reply:
Why?

Reply:
Reverted to 0.8 opacity on [a6b16fd](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/a6b16fdf489a4652038ac614da23fe5485e62020)